### ISSUE TITLE
forge init fails with fatal: not a git repository (or any of the parent directories): .git

### ISSUE THREAD

**Issue by czepluch** [foundry-rs/foundry#3733]
### Component

Forge

### Have you ensured that all of these are up to date?

- [X] Foundry
- [X] Foundryup

### What version of Foundry are you on?

forge 0.2.0 (cb925b1 2022-11-21T00:09:19.970948Z)

### What command(s) is the bug in?

forge init

### Operating System

macOS (Apple Silicon)

### Describe the bug

I've used `forge init` several times before, but when I just ran `foundryup` to update to the latest version and tried to init a new project I got the following error:
```
forge init foobar
Initializing /dev/eth/foobar...
Installing forge-std in "/dev/eth/foobar/lib/forge-std" (url: Some("https://github.com/foundry-rs/forge-std"), tag: None)
Error:
fatal: not a git repository (or any of the parent directories): .git
```

**Comment by wp-lai** [comment]
I'm having the same issue.

**Comment by rkrasiuk** [comment]
@mattsse seems like there is one more issue with checking out the latest tag
<img width="1531" alt="Screenshot 2022-11-22 at 13 46 20" src="https://user-images.githubusercontent.com/25429261/203306269-9a8acc28-409f-425b-8d41-39e17ee09c3d.png">

**Comment by Robsonsjre** [comment]
+1 (Having the same issue)

**Comment by mattsse** [comment]
woops, sorry about that,

forgot to set the correct dir for the git process: should be fixed here #3750
